source/user-manual/api/configuration.rst

```diff
@@ -54,9 +54,18 @@ Here are all the available settings for the ``api.yaml`` configuration file. For
         block_time: 300
         max_request_per_minute: 300
 
+     remote_commands:
+        localfile:
+           enabled: yes
+           exceptions: []
+
+       wodle_command:
+          enabled: yes
+          exceptions: []
+
 .. warning::
 
-    If running a cluster, the master will NOT send its local Wazuh API configuration file to the workers. Each node provides its own Wazuh API. If the configuration file is changed in the master node, the user should manually update the workers Wazuh API configuration in order to use the same one. Take care of not overwriting the IP and port in the local configuration of each worker. The Wazuh API endpoint :api-ref:`PUT /cluster/api/config <operation/api.controllers.cluster_controller.put_api_config>` can be used to change any or all of the Wazuh API configuration files in the cluster nodes.
+    If running a cluster, the master will NOT send its local Wazuh API configuration file to the workers. Each node provides its own Wazuh API. If the configuration file is changed in the master node, the user should manually update the workers Wazuh API configuration in order to use the same one. Take care of not overwriting the IP and port in the local configuration of each worker.
```

Comment:
I would suggest changing "Take care of not overwriting" to "Be sure to not overwrite"

Reply:
Done.